### PR TITLE
Fix rendering of global tracking script

### PR DESCRIPTION
**Pull request by DanielCollins**

### CODE REVIEW

File: python/util/google.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 <script async src="https://www.googletagmanager.com/gtag/js?id={}"></script>
 <script>
   window.dataLayer = window.dataLayer || [];
-  function gtag(){dataLayer.push(arguments);}
+  function gtag(){{dataLayer.push(arguments);}}
```
Comment (s3341458) [inline python/util/google.py]: do not really understand why this change is needed?

Comment (DanielCollins) [reply python/util/google.py]: this string used as a format string. `{a}` is a format specifier for `a`.
